packages/modules/auth/integration-tests/__tests__/auth-module-service/index.spec.ts

```diff
@@ -36,6 +37,25 @@ moduleIntegrationTestRunner({
         })
       })
 
+      it(`should export the appropriate linkable configuration`, () => {
+        const linkable = Module(Modules.AUTH, {
+          service: AuthModuleService,
+        }).linkable
+
+        expect(Object.keys(linkable)).toEqual(["authIdentity"])
+
+        linkable.authIdentity.toJSON = undefined
+
+        expect(linkable.authIdentity).toEqual({
+          id: {
+            linkable: "auth_identity_id",
+            primaryKey: "id",
+            serviceName: "auth",
```

Comment:
question: It looks like the `serviceName` is actually the module name. Is that correct @adrien2p, or just a coincidence?

Reply:
yes exactly it is as we are using the `Modules` enum, I could rename it but it was to be aligned with all the config and mention we have in the joinerConfig, link config etc. The service name wording came from the micro service real

Reply:
it is also the name we choosen for the utils such as joiner config